doc/advanced_configuration.rst

```diff
@@ -24,6 +24,43 @@ the sphinx configuration dictionary:
 
 Keep in mind that both lists have to be of the same length.
 
+.. _build_pattern:
+
+Building examples matching a pattern
+====================================
+
+By default, Sphinx-Gallery execute only examples beginning with ``plot``. However,
+if this naming convention does not suit your project, you can modify this pattern
+in your Sphinx ``conf.py``. For example:
+
```

Comment:
I think as we now match to directory and filename so that we have a rich set of options, is relevant  to Introduce that. That also introduces why you are putting the leading slash. Which is only valid for *nix systems, certainly put it as the default is written at gen_gallery.rst